CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.0.8
```

Comment:
Quick note: changes that are being merged into develop will always be categorized under the `# TBD` (to be determined) section. Only when you cut the release will `TBD` turn into the actual version number. The rationale for this is that the version number will actually change depending on the changes to be released: e.g. if there are no breaking changes, we'll bump the `Z` so the new version will be `X.Y.Z+1`. If there are breaking changes, we'll bump `Y` so that it will become `X.Y+1.0`

Reply:
Yes! you are right! I forgot to add the TBD, Now I understand this very clear.